src/App.js

```diff
@@ -1,16 +1,42 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
+import { InitialCountForm } from './components'
 import './styles/app.css'
+import PropTypes from 'prop-types'
 
 class App extends Component {
+  state = { initialCount: 0, shouldUpdate: false }
+
+  myCallback = (initialCount, shouldUpdate) => {
+    this.setState({ initialCount: initialCount, shouldUpdate: shouldUpdate })
```

Comment:
You can use a shorthand notation here if the key and value you're setting have the same name.
```jsx
this.setState({ initialCount, shouldUpdate })
```